ssg/build_yaml.py

```diff
@@ -1499,8 +1502,12 @@ def _process_rules(self):
             self.loaded_group.add_rule(rule)
 
             if self.loaded_group.platforms:
-                rule.inherited_platforms += self.loaded_group.platforms
+                rule.inherited_platforms.update(self.loaded_group.platforms)
 
+            if self.loaded_group.inherited_platforms:
+                rule.inherited_platforms.update(self.loaded_group.inherited_platforms)
+
+            
```

Comment:
Same thing here, the two `ifs` are handling the platforms a rule inherits from the group it gets inserted into.
It seems to me that these two conditionals should be moved to the Group's `add_rule()` method.